src/main/scala/io/ino/sbtpillar/Plugin.scala

```diff
@@ -178,14 +182,21 @@ object Plugin extends sbt.Plugin {
       }
     }
 
-    def migrate(migrationsDir: File, url: CassandraUrl, defaultConsistencyLevel: ConsistencyLevel, logger: Logger): Unit = {
+    def migrate(migrationsDir: File, url: CassandraUrl, defaultConsistencyLevel: ConsistencyLevel, logger: Logger): Unit =
```

Comment:
No longer used (as `migrate(session: Session...)` is not used as well), therefore can be removed. And then you can rename `migrateAll` to `migrate` :-)

I don't consider this change as compatibility breaking, because this is not a library but an sbt plugin, so that Tasks/Settings are what has to be kept compatible (in the case you've left these methods untouched due to compatibility concerns).